### PR TITLE
refactor: icon dialog&lightbox

### PR DESCRIPTION
**Pull request by kyubisation**

### CODE REVIEW

File: src/angular-business/dialog/dialog-container/dialog-container.component.ts [hunk 1]
```diff
@@ -47,56 +46,59 @@ export function throwDialogContentAlreadyAttachedError() {
   // Using OnPush for dialogs caused some G3 sync issues. Disabled until we can track them down.
   changeDetection: ChangeDetectionStrategy.Default,
   animations: [DIALOG_ANIMATIONS.slideDialog],
+  host: {
+    class: 'sbb-dialog-container',
+    tabindex: '-1',
+    'aria-modal': 'true',
+    '[attr.id]': 'id',
+    '[attr.role]': 'config.role',
```
Comment (jeripeierSBB) [inline src/angular-business/dialog/dialog-container/dialog-container.component.ts:54]: I thought this does not work. Did you test in a browser?

File: src/angular-public/lightbox/lightbox/lightbox-container.component.ts [hunk 2]
```diff
@@ -47,56 +47,59 @@ export function throwLightboxContentAlreadyAttachedError() {
   // Using OnPush for dialogs caused some G3 sync issues. Disabled until we can track them down.
   changeDetection: ChangeDetectionStrategy.Default,
   animations: [LIGHTBOX_ANIMATIONS.slideLightbox],
+  host: {
+    class: 'sbb-lightbox-container',
+    tabindex: '-1',
+    'aria-modal': 'true',
+    '[attr.id]': 'id',
+    '[attr.role]': 'config.role',
```
Comment (jeripeierSBB) [inline src/angular-public/lightbox/lightbox/lightbox-container.component.ts:55]: same question with role

File: src/angular-public/lightbox/lightbox/lightbox-content.ts [hunk 3]
```diff
@@ -125,14 +129,15 @@ export class LightboxHeaderComponent implements OnInit {
 
 @Directive({
   selector: `[sbbLightboxTitle]`,
+  host: {
+    class: 'mat-lightbox-title',
```
Comment (jeripeierSBB) [inline src/angular-public/lightbox/lightbox/lightbox-content.ts:133]: really? ;)

Comment (kyubisation) [reply src/angular-public/lightbox/lightbox/lightbox-content.ts:133]: Oops

File: src/angular-business/dialog/dialog-container/dialog-container.component.ts (hunk 1 above)
Comment (kyubisation) [reply src/angular-business/dialog/dialog-container/dialog-container.component.ts:54]: This certainly works. however 'attr.role' probably does not work

File: src/angular-public/lightbox/lightbox/lightbox-content.ts (hunk 3 above)
Comment (kyubisation) [inline src/angular-public/lightbox/lightbox/lightbox-content.ts:133]:
```suggestion
    class: 'sbb-lightbox-title',
```

File: src/angular-business/dialog/dialog-container/dialog-container.component.ts (hunk 1 above)
Comment (jeripeierSBB) [reply src/angular-business/dialog/dialog-container/dialog-container.component.ts:54]: sorry I did remember the case i wrote 'attr.role': '...'. But yes, [attr.role] is working for sure :)

**Pull request merged**